### PR TITLE
fix(TW-659): Squelch errors when given an invalid ID format for a model update

### PR DESCRIPTION
**Pull request by uptickmetachu**
https://uptickhq.atlassian.net/jira/software/projects/TW/boards/69/backlog?label=sentry_escalated&selectedIssue=TW-659

Associated error.

Here we are checking for the error... and returning it as a user visible error instead of raising it out as an uncaught exception.

### CODE REVIEW

**Comment by uptickmetachu** [comment]
image

**Comment by uptickmetachu** [comment]
Tested locally. Works as expected.

File: djangomodelimport/__init__.py [hunk 1]
```diff
@@ -7,4 +7,4 @@
 )  # noqa
 from .resultset import ImportResultRow, ImportResultSet  # noqa
 
-__version__ = "0.5.1"
+__version__ = "0.6.2"
```
Comment (jarekwg) [inline djangomodelimport/__init__.py:10]: Why the strange bump? :D

Comment (jarekwg) [reply djangomodelimport/__init__.py:10]: ah dw, allg.

Comment (bayangan1991) [reply djangomodelimport/__init__.py:10]: Think I just missed it last time, oops

**Pull request merged**